test/PowerShellEditorServices.Test.Host/LanguageServerTests.cs

```diff
@@ -825,7 +825,9 @@ public class LanguageServerTests : ServerTestsBase, IAsyncLifetime
             Assert.StartsWith("5.", versionDetails.Version);
             Assert.StartsWith("5.", versionDetails.DisplayVersion);
             Assert.Equal("Desktop", versionDetails.Edition);
-            Assert.Equal("x86", versionDetails.Architecture);
+
+            string expectedArchitecture = (IntPtr.Size == 8) ? "x64" : "x86";
```

Comment:
This will eventually be a problem when we want to run the tests on Non-Windows because on macOS `x64` is represented as `x86_64`

Reply:
but that can be dealt with later. I see we have an Assert.StartsWith just a few lines up that mentions version 5 so we'll have to deal with that too.

Reply:
The PowerShellVersionDetails.Architecture property is an enum with only these values: `Unknown`, `x86`, `x64`.  So I think it is OK represent a 64-bit macOS Architecture as x64.  Besides the version number, the only other place this info is used is to send back to the extension.  And it does get logged.  Even the log chooses to represent the Architecture.x64 val as simply `64-bit`:
```
        Version: Microsoft Windows NT 10.0.16299.0
        Arch:    64-bit
```
In this regard, I like the IntPtr.Size test better than looking at env vars.